### PR TITLE
Added .project(bazel) and IDEA to .gitignore

### PR DESCRIPTION
**Pull request by maximn**

### CODE REVIEW

**Comment by damienmg** [comment]
Shouldn't that be in user specific .gitignore?

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 /bazel-*
+/.project
+
```
Comment (ittaiz) [inline .gitignore]: can you please remove this empty line?

**Comment by ittaiz** [comment]
good question. Bazel's own .gitignore file adds these and much more.
.project for example is something which is unrelated to @maximn as a user but is a by-product of some of the code of this lib

**Comment by damienmg** [comment]
Ok make sense, maybe add comment to each section, e.g.:

```
# Bazel output directories
bazel-*
# IDE generated files
/idea
/.project
```
?

**Comment by ittaiz** [comment]
update- actually .project seems like bazel-intellij related but still both are part of Bazel's own gitignore.
@damienmg wdyt?

**Comment by damienmg** [comment]
As it make sense to have that as a default ignore to everybody that's fine by me.

**Comment by ittaiz** [comment]
sounds good. only comment is that the bazel one needs to be `/bazel-*` in case a sub directory is called `bazel-foo`

**Comment by damienmg** [comment]
Commit LGTM, there is just a missing newline at the end.

**Comment by ittaiz** [comment]
@philwo I think we are at somewhat of an impasse with respect to PRs and maybe you have relevant inputs.
This PR started from master before #36 was merged and so I saw that I *have* to "update from master" as this is required.
On the other hand when I do so I get a failure from the cla bot :(
WDYT?

**Comment by maximn** [comment]
added the new line.  @damienmg Why is it needed?

**Comment by philwo** [comment]
@maximn Just for style guide reasons. We take care in Bazel's code to not have trailing whitespace in lines and that each file ends with a newline.

**Comment by ittaiz** [comment]
@philwo interesting to see if asking pull request author to click the button can prevent this from happening.
In any case I'd appreciate you checking with the cla-bot maintainers about this since it surfaces in many of the projects

**Pull request merged**